pycm/generate_random_data.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+"""This file contains a function to generate a random confusion matrix."""
+import numpy as np
+from enum import Enum
+
+
+class Scenario(Enum):
+    """
+    Enum to represent different scenarios for generating class percentages.
+
+    - UNIFORM: All classes have equal representation.
+    - MAJORITY_CLASS: One class has a majority representation, others are minority.
+    - MINORITY_CLASS: One class has a minority representation, others are majority.
```

Comment:
I'd suggest changing them to `Only one class has a majority/minority representation`, what it is right now with `others are minority/majority` can be confusing.